recipes/python/retail/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN /usr/bin/python3.5 -m pip install -U numpy
 RUN /usr/bin/python3.5 -m pip install -U pandas
 RUN /usr/bin/python3.5 -m pip install -U sklearn
 RUN /usr/bin/python3.5 -m pip install -U scipy
-RUN /usr/bin/python3.5 -m pip install -U fastparquet
+RUN /usr/bin/python3.5 -m pip install -U fastparquet==0.2.1
```

Comment:
if we left without version, it will install latest, but latest version not working with new schemas. I had to specify a bit older version and everything is working fine.

Reply:
Is this profile data?  If not, `0.1.4` should work.

Reply:
Is there any reason all these install commands are not in one line?  There are optimizations to gain if so.  
Also, we should consider indicating a version for each dependency for consistency and semi-repeatability.

Reply:
Hi @paulhsiung 
If we left to 0.1.4 version we are getting  : ImportError: fastparquet >= 0.2.1 is required for parquet support
If we left without version it will install latest and the latest not work too. but with version 0.2.1 is running without problems.
I'm not sure what you meant with profile data?
If you mean it is profile or time series, it is time series.

Reply:
@rjrussell77 
I'm not sure. That is the existing code. I saw the same approach at various Dockerfiles in our projects.

Reply:
As long as `0.2.1` works for you, that is great.
I believe this error (`ImportError: fastparquet >= 0.2.1 is required for parquet support`) comes from the latest pandas

Reply:
If you have time, when you run and test your code, can you note the versions of pandas, sklearn, and scipy then add those versions to the RUN commands (e.g. `RUN /usr/bin/python3.5 -m pip install -U pandas==1.0.0`).  Otherwise, we have a moving target when users actually do this since those libraries can change and then we don't know if they all work together.

Reply:
Yes, I will write that versions down, and let you know.

Reply:
@rjrussell77 
Here are the current working versions of the dependencies. 
From some reason it throws error when specify version for sklearn but from logs while building I can see this version installed "Successfully installed scikit-learn-0.20.3"

Reply:
Our data science team looked into this and recommend, `pip install -U scikit-learn` See here: https://scikit-learn.org/stable/install.html

Reply:
Added back Dockerfile commands since I'm getting error while building with new changes. Also we agree that is good to know exact versions of the dependencies, so I've added them all.